--- a/Makefile	2021-04-08 23:28:39.000000000 +0200
+++ b/Makefile	2021-04-28 09:34:13.676792000 +0200
@@ -107,8 +107,8 @@
 $(foreach D,$(FFLIBS),$(eval $(call DOSUBDIR,lib$(D))))
 
 include $(SRC_PATH)/fftools/Makefile
-include $(SRC_PATH)/doc/Makefile
-include $(SRC_PATH)/doc/examples/Makefile
+#include $(SRC_PATH)/doc/Makefile
+#include $(SRC_PATH)/doc/examples/Makefile
 
 libavcodec/avcodec.o libavformat/utils.o libavdevice/avdevice.o libavfilter/avfilter.o libavutil/utils.o libpostproc/postprocess.o libswresample/swresample.o libswscale/utils.o : libavutil/ffversion.h
 
@@ -174,7 +174,7 @@
 build: all alltools examples testprogs
 check: all alltools examples testprogs fate
 
-include $(SRC_PATH)/tests/Makefile
+#include $(SRC_PATH)/tests/Makefile
 
 $(sort $(OUTDIRS)):
 	$(Q)mkdir -p $@
